### DISCUSSION TITLE
Ignore invalid errors

### DISCUSSION THREAD

**Discussion by jeremyhagan** [microsoft/vscode-azurearmtools#1766] [Q&A] [answered]
Can someone point me towards how to get the extension to ignore invalid schema errors. At the moment it doesn't support Filter and so I'd like to tell it to ignore specific invalid function names like filter and lambda

**Comment by StephenWeatherford** [comment, marked as answer]
Sorry, didn't see this.  Currently there's no "official" way to do this. However, lambda and filter will be in the next released version, expected early December.

In the meantime, the *unofficial* way is to edit this file:
```
C:\Users\<user>\.vscode{-insiders}\extensions\msazurermtools.azurerm-vscode-tools-0.15.<version>\assets\ExpressionMetadata.json
```
That controls the validation and intellisense for template functions.  Here's the updated one that has lambda/etc:
[ExpressionMetadata.zip](https://github.com/user-attachments/files/17876667/ExpressionMetadata.zip)

Note that any changes you make will be overridden when the extension is updated.

**Reply by jeremyhagan** [reply]
Thanks!
